# v1.0 last modified on 23/02/2022
# message types p0, pf, v0, vf, to, tf added 
# Kumar Sindhurakshit 23/02/2022

float64 p0
float64 pf
float64 v0
float64 vf
float64 t0
float64 tf

